cypress/storybook/storybook.spec.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable jest/expect-expect */
```

Comment:
sure... but if we're setting up our jest regex to exclude these files, then our linter should also exclude these files ideally.

Reply:
yeah meant to remove that - am still unsure why eslint is still saying that the test has no assertions - but it runs and passes either way